pkg/client/client.go

```diff
@@ -992,6 +992,18 @@ func GetPrimaryProxy(url string) (string, error) {
 	return smap.ProxySI.DirectURL, nil
 }
 
+func UnregisterTarget(url, sid string) error {
+	err := HTTPRequest("DELETE", url+"/"+dfc.Rversion+"/"+dfc.Rcluster+"/"+"daemon"+"/"+sid, nil)
+	// FIXME: do something better than sleep
+	time.Sleep(time.Second * 3)
```

Comment:
I do not think it should be in client. In my opinion it should better be in test where it calls the client library

Reply:
While driving home, I thought about this way to replace waiting with random time. It seems sleep is done to be sure that all nodes have updated Smap. Maybe replace Sleep with a function that reads Smap from alive proxies and targets and compares their Smap version? And it loops until timeout or versions are the same